src/app/events/events.page.html

```diff
@@ -9,7 +9,7 @@
   </div>
   <app-title-fades [color]="titleColor" [title]="titleText" />
   <ng-container *ngFor="let event of eventsData">
-    <app-event [event]="event" />
+    <app-event [event]="event" [eventId] = "deleteEvents"/>
```

Comment:
what is this "[eventId] = "deleteEvents" " ?

Reply:
I remove it and made it better